.gitignore

```diff
@@ -71,6 +71,7 @@ __pycache__/
 /Code/GraphMol/test_data/prop.pkl
 
 /Projects/DbCLI/testData/bzr/
-
+External/INCHI-API/*
```

Comment:
I use `~/.gitignore_global` to keep directories like this or files that are created by the IDE from appearing in a list of potential files to be added to github. For example, I have these two lines in it.
```
/External/AvalonTools/*
/External/INCHI-API/*
```